default/python/AI/AIDependencies.py

```diff
@@ -20,6 +20,8 @@
     my_industry = AIDependencies.INDUSTRY_PER_POP * my_population
 """
 import freeOrionAIInterface as fo  # interface used to interact with FreeOrion AI client  # pylint: disable=import-error
+from common.configure_logging import convenience_function_references_for_logger
+(debug, info, warn, error, fatal) = convenience_function_references_for_logger(__name__)
```

Comment:
It is look very ugly, and uncommon.
`(debug, _, _, _, _) = convenience_function_references_for_logger(__name__)`

I didn't ever use python logging as raw code, only as part of frameworks.  I don't know how to do it properly.

I see next way:
`from freeorion_logging import debug`

Log setup will be done in freeorion_logging module (execute on import).  Have import from it -> have logging setuped, no import no setup.  

This function should have same signature as logging `debug`,   Just save link it in module. `debug = logging.debug`